src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <!-- #region HEAD -->
+    <header class="header container">
+
+      <a href="#home" class="logo">
+        <img src="./images/logo.png"
+             alt="logo"
+             class="logo__img">
+      </a>
```

Comment:
no need to add blanks between parent and child and check your code style
```suggestion
    <header class="header container">

      <a href="#home" class="logo">
        <img
          src="./images/logo.png"
          alt="logo"
          class="logo__img"
        >
      </a>
```